src/_guides/language/language-tour.md

```diff
@@ -1136,25 +1136,31 @@ The <code>=> <em>expr</em></code> syntax is a shorthand for
 <code>{ return <em>expr</em>; }</code>. The `=>` notation
 is sometimes referred to as _arrow_ syntax.
 
-<div class="alert alert-info" markdown="1">
-**Note:**
-Only an *expression*—not a *statement*—can appear between the arrow
-(=\>) and the semicolon (;). For example, you can’t put an [if
-statement](#if-and-else) there, but you can use a [conditional
-expression](#conditional-expressions).
-</div>
+<aside class="alert alert-info" markdown="1">
+  **Note:**
+  Only an *expression*—not a *statement*—can appear between the arrow
+  (=\>) and the semicolon (;). For example, you can’t put an [if
+  statement](#if-and-else) there, but you can use a [conditional
+  expression](#conditional-expressions).
+</aside>
 
-A function can have two types of parameters: required and optional. The
-required parameters are listed first, followed by any optional
-parameters. Named optional parameters can also be marked as `@required`.
-See the next section for details.
+A function can have two types of parameters: _required_ and _optional_.
+The required parameters are listed first, followed by any optional parameters.
+Optional parameters can be _named_ or _positional_.
 
+<aside class="alert alert-info" markdown="1">
+  **Note:**
+  Some APIs — notably [Flutter][] widget constructors —
+  use only optional named parameters,
+  even for parameters that are mandatory.
```

Comment:
use only optional named parameters, even for params that are mandatory ?   This sentence confuses me…

Reply:
That's because it's inherently confusing. :/

Reply:
I've changed "optional named" to "named" here.